### ISSUE TITLE
[Bug] Incorrect Argument Passing in softmax_fwd_kernel Call

### ISSUE THREAD

**Issue by p81sunshine** [fla-org/flash-linear-attention#187]
### Checklist

- [x] I have checked [FAQs](https://github.com/fla-org/flash-linear-attention/blob/main/FAQs.md) and existing issues for similar problems
- [x] Please report this bug in English to ensure wider understanding and support

### Describe the Bug

In the file flash-linear-attention/fla/ops/abc, the softmax_fwd_kernel function is called with the following arguments:

```python
softmax_fwd_kernel[grid](
    ok, p,
    s.stride(1), s.stride(2), s.stride(3),
    T=T, S=M, BT=BT
)
```

However, the definition of softmax_fwd_kernel (flash-linear-attention/fla/ops/utils/softmax.py) is:

```python
def softmax_fwd_kernel(
    x,
    p,
    D: tl.constexpr,
    B: tl.constexpr
):
```

It appears that the call is passing extra arguments (s.stride(1), s.stride(2), s.stride(3), T=T, S=M, BT=BT) that are not defined in the function signature. This can cause the ops in flash-linear-attention/fla/ops/abc/chunk.py can not be compiled. 

<img width="699" alt="Image" src="https://github.com/user-attachments/assets/af7eb4d4-c5bc-4ad8-8c42-200217922c9d" />

### Steps to Reproduce the Bug

import torch
from fla.ops import fused_chunk_based
from fla.ops.abc import chunk_abc
q,k,v = torch.randn(3, 2, 16, 16).cuda(), torch.randn(3, 2, 16, 16).cuda(), torch.randn(3, 2, 16, 16).cuda()
s = torch.randn(3, 2, 16, 16).cuda()
s[0,0,2,0] = 908.5
s[0,0,0,0] = 0
s[0,0,3,0] = 0
s[0,0,1,0] = 0
q.requires_grad_()
k.requires_grad_()
v.requires_grad_()
s.requires_grad_()
o = chunk_abc(q,k,v,s)

### Expected Behavior

The kernel can not be compiled. 

### Environment Information

1. Torch:
2. Triton:
